spring-core/src/main/java/org/springframework/util/PlaceholderParser.java

```diff
@@ -443,34 +443,30 @@ record SimplePlaceholderPart(String text, String key, @Nullable String fallback)
 
 		@Override
 		public String resolve(PartResolutionContext resolutionContext) {
-			String resolvedValue = resolveToText(resolutionContext, this.key);
```

Comment:
Please revert all the formatting change as it makes it harder for us to see what you've changed.